pl_examples/basic_examples/mnist.py

```diff
@@ -17,7 +17,12 @@
 import pytorch_lightning as pl
 from torch.nn import functional as F
 from torch.utils.data import DataLoader, random_split
-from tests.base.datasets import MNIST
+
+try:
```

Comment:
for examples, we expect that `torchvision` is installed

Reply:
on the other hand, `pl_examples` is installed with PL but not tests, so if someone has installed PL and missing `torchvision` it will fail also because tests are missing